modules/rke/harvester/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "rke2_server1_public_dns" {
```

Comment:
The name needs to be public_ip for all of them, or else this will fail upon rebase.